tests/monero.rs

```diff
@@ -334,6 +334,7 @@ async fn monero_syncer_address_test() {
                 lifetime: blocks + 1,
                 addendum: addendum_1.clone(),
                 include_tx: Boolean::True,
+                filter: TxFilter::Incoming,
```

Comment:
no tests of `TxFilter::Outgoing`, but this is fine since we don't need to watch outgoing txs with farcaster (we could check whether counterparty sweeps the xmr address, but not needed for the protocol).